en/deploy-on-azure-aks.md

```diff
@@ -0,0 +1,582 @@
+---
+title: Deploy TiDB on Azure AKS
+summary: Learn how to deploy a TiDB cluster on Azure Kubernetes Service (AKS).
+aliases: ['/docs/tidb-in-kubernetes/dev/deploy-on-azure-aks/']
+---
+
+# Deploy TiDB on Azure AKS
+
+This document describes how to deploy a TiDB cluster on Azure Kubernetes Service (AKS).
+
+To deploy TiDB Operator and the TiDB cluster in a self-managed Kubernetes environment, refer to [Deploy TiDB Operator](deploy-tidb-operator.md) and [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md).
+
+## Prerequisites
+
+Before deploying a TiDB cluster on Azure AKS, make sure the following requirements are satisfied:
+
+* Install [Helm 3](https://helm.sh/docs/intro/install/) for deploying TiDB Operator.
+* [Create a Kubernetes cluster](https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-deploy-cluster) and install and configure `az cli`.
+
+    > **Note：**
+    >
+    > To verify whether AZ CLI is configured correctly, run the `az login` command. If login with account credentials succeeds, AZ CLI is configured correctly. Otherwise, you need to re-configure AZ CLI.
+
+* [Create a new cluster that can use Ultra disks](https://docs.microsoft.com/en-us/azure/aks/use-ultra-disks).
+
+* Install [Helm 3](https://helm.sh/docs/intro/install/): used for deploying TiDB Operator.
+* Complete all operations in [Create a Kubernetes cluster](https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-deploy-cluster) and [Create a new cluster that can use Ultra disks](https://docs.microsoft.com/en-us/azure/aks/use-ultra-disks).
```

Comment:
```suggestion
```

Reply:
These two item lists are redundant and should be removed.

Reply:
fixed